src/storage/gcloud-storage/storage.ts

```diff
@@ -0,0 +1,111 @@
+import type { Storage } from '@google-cloud/storage';
+import type { Account } from 'account/account.entity';
+import type { AccountService } from 'account/account.service';
+import { type Result, error, getError, isError, ok } from 'core/result';
+import type { Context } from 'hono';
+import { v4 as uuidv4 } from 'uuid';
+
+const ALLOWED_IMAGE_TYPES = [
+    'image/jpg',
+    'image/jpeg',
+    'image/png',
+    'image/webp',
+];
```

Comment:
We can also allow image/gif or image/svg? But doesn't seem necessary for profile/cover images